app/assets/stylesheets/common/base/search.scss

```diff
@@ -97,7 +93,14 @@
       }
 
       button {
-        margin: 0 0.5em 0.5em 0;
+        margin: 0 0.5em 0 0;
+        border: 1px solid var(--primary-low-mid);
```

Comment:
This feels weird, having a border color and then hiding it on hover. I would prefer leaving this to default.